statics/js/components/workflow.js

```diff
@@ -88,6 +88,8 @@ Vue.component('workflow-params', {
   methods: {
 
     submit: function() {
+      this.display.result = true;
+      this.result.value = undefined;
       var self = this;
```

Comment:
Hello
Could you please wrap these two lines into one function and call it like toggleReadyStatus ?
It's better to turn on/off flags in setters,

Reply:
done